cannot build sphinx document

I am trying to build the documentation sphinx document but I cannot make it compile. The error I get is (jwst_dev) bash-3.2$ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.7.6

Configuration error:
There is a programable error in your configuration file:

Traceback (most recent call last):
  File "/Users/rdiaz/anaconda3/envs/astroconda/lib/python3.5/site-packages/sphinx/config.py", line 161, in __init__
    execfile_(filename, config)
  File "/Users/rdiaz/anaconda3/envs/astroconda/lib/python3.5/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "conf.py", line 19, in <module>
    import stsci_rtd_theme
ImportError: No module named 'stsci_rtd_theme'

Makefile:49: recipe for target 'html' failed
make: *** [html] Error 2
(jwst_dev) bash-3.2$ 

I downloaded a version of the stsci_rtd_theme package and installed it but still get the same error. Can someone help with this? I have spent already a lot of time trying to make it work.

--- a/README.md
+++ b/README.md
@@ -69,6 +69,11 @@ Documentation (built daily from `master`) is available here:
 
 https://jwst-pipeline.readthedocs.io/en/latest/
 
+One can clone this repository and build the documentation with
+
+    pip install sphinx_rtd_theme stsci_rtd_theme sphinx_automodapi
+    python setup.py build_sphinx
+
 
 Contributions and Feedback
 --------------------------

--- a/README.md
+++ b/README.md
@@ -105,12 +105,11 @@ Software vs DMS build version map
 Unit Tests
 ----------
 
-Unit tests can be run via `pytest`.  All tests need a the `ci_watson` pytest plugin to run.  We also recommend using `pytest-xdist` so you can run them in parallel.
+Unit tests can be run via `pytest`.  All tests need a the `ci_watson` pytest plugin to run.
 
-    pip install ci-watson pytest-xdist
-    pytest -n <cores>
+    pip install requests_mock ci_watson
+    pytest jwst
 
-where `cores` is the number of cores you'd like to use on your machine for the tests.
 
 Regression Tests
 ----------------
@@ -123,17 +122,14 @@ https://boyle.stsci.edu:8081/job/RT/job/JWST/
 
 The test builds start at 11am and 6pm local Baltimore time every day on jwcalibdev.
 
-To run the regression tests on your local machine, you need the `ci_watson` pytest plugin as above.  Then `rsync` or `scp` the input and comparison data locally
+To run the regression tests on your local machine, you need the `ci_watson` pytest plugin as above.  Then set the environment variable TEST_BIGDATA to either our Artifactory server or the local upload cache on central store (STSci staff members only)
 
-    rsync -av <username>@jwcalibdev:/data4/jwst_test_data /my/local/path/
+    export TEST_BIGDATA=https://bytesalad.stsci.edu/artifactory/
 
-set the environment variable TEST_BIGDATA to this location
+or
 
-    export TEST_BIGDATA=/my/local/path/jwst_test_data/
+    export TEST_BIGDATA=/grp/jwst/ssb/jwst_test_data
 
 and then run the tests in the repository
 
-    cd /path/to/jwst/jwst/tests_nightly/general
-    pytest --bigdata .
-
-
+    pytest --bigdata jwst/tests_nightly/general

